### ISSUE TITLE
Docker-tiup-playground tiflash error : version `GLIBC_PRIVATE' not found (on macos apple m1)

### ISSUE THREAD

**Issue by dulao5** [ti-click/docker-tidb-playground#2]
```
docker-tidb-playground-tidb-1  | /root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash: /lib/aarch64-linux-gnu/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash)

docker compose exec tidb bash :

root@2ced9528ce47:/# cd
root@2ced9528ce47:~# tail /root/.tiup/data/devenv/tiflash-0/tiflash.log
/root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash: /lib/aarch64-linux-gnu/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash)

```

**Comment by dulao5** [comment]
```
root@2ced9528ce47:/# ls -l /lib/aarch64-linux-gnu/libpthread.so.0
-rw-r--r-- 1 root root 13736 Jul  7  2022 /lib/aarch64-linux-gnu/libpthread.so.0
root@2ced9528ce47:/# md5sum /lib/aarch64-linux-gnu/libpthread.so.0
77366f227d291adee8137ea9572d961c  /lib/aarch64-linux-gnu/libpthread.so.0

root@2ced9528ce47:~# ldd /root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash
/root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash: /lib/aarch64-linux-gnu/libpthread.so.0: version `GLIBC_PRIVATE' not found (required by /root/.tiup/components/tiflash/v6.1.5/tiflash/tiflash)
        linux-vdso.so.1 (0x0000ffffa227d000)
        libc++.so.1 => /root/.tiup/components/tiflash/v6.1.5/tiflash/libc++.so.1 (0x0000ffffa2140000)
        libc++abi.so.1 => /root/.tiup/components/tiflash/v6.1.5/tiflash/libc++abi.so.1 (0x0000ffffa20d0000)
        libtiflash_proxy.so => /root/.tiup/components/tiflash/v6.1.5/tiflash/libtiflash_proxy.so (0x0000ffff9f160000)
        libdl.so.2 => /lib/aarch64-linux-gnu/libdl.so.2 (0x0000ffff9f140000)
        librt.so.1 => /lib/aarch64-linux-gnu/librt.so.1 (0x0000ffff9f120000)
        libm.so.6 => /lib/aarch64-linux-gnu/libm.so.6 (0x0000ffff9f080000)
        libpthread.so.0 => /lib/aarch64-linux-gnu/libpthread.so.0 (0x0000ffff9f060000)
        libgcc_s.so.1 => /lib/aarch64-linux-gnu/libgcc_s.so.1 (0x0000ffff9f030000)
        libc.so.6 => /lib/aarch64-linux-gnu/libc.so.6 (0x0000ffff9ee80000)
        /lib/ld-linux-aarch64.so.1 (0x0000ffffa2244000)
```

**Comment by dulao5** [comment]
fixed by #3
